applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -32,7 +32,7 @@ config DESKTOP_BLE_USE_DEFAULT_ID
 
 config DESKTOP_BLE_PEER_CONTROL
 	bool "Enable interaction with peer configuration"
-	depends on !DESKTOP_BUTTONS_NONE
+	depends on CAF_BUTTONS
```

Comment:
`depends on CAF_BUTTON_EVENTS`

Reply:
Similar as for `Kconfig.click_detector`.